connman/include/inet.h

```diff
@@ -56,11 +56,12 @@ int connman_inet_set_ipv6_address(int index,
 int connman_inet_clear_ipv6_address(int index,
 					struct connman_ipaddress *ipaddress);
 int connman_inet_add_ipv6_network_route(int index, const char *host,
-					const char *gateway, unsigned char prefix_len);
+					const char *gateway,
+					unsigned char prefix_len, short metric);
```

Comment:
Again, if this is a public interface to the external plugins (looks like it - these functions are exported from connman and the header is a part of connman-devel package), function signatures must not change over time.

Reply:
afaict previously fixed metric=1 value waws . so perhaps api compatibility could be retained by adding a new function xxx_with_metric() and making existing connman_inet_add_ipv6_network_route() call it with metric value 1?

Reply:
Changed to add a new function there, so everything uses the `_with_metric()` keeping all other remaining function uses the same: 6042574547fa41cee13f3715b5575c323a3d4e48 + 30cf0fa36c0fa42d3befc25eaa09c21fde965ac6 + 6d1254074d63993fd705d0a4937f5047b5fdc06c + 37018906609f032c16734686a054efb2b592c465